extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpConfiguration.java

```diff
@@ -289,6 +289,14 @@ public class HttpConfiguration {
     @ConfigItem
     public Map<String, FilterConfig> filter;
 
+    /**
+     * This will decorate the stacktrace to show the line in the code that cause the exception
+     * By default, this is enabled in Dev Mode and disabled in all other modes. Making this true will enable it in all mode
+     * and making this false will disable this in all modes.
+     */
+    @ConfigItem
+    public Optional<Boolean> decorateStacktraces;
```

Comment:
Not fully convinced its good to allow enable this in prod up front as it will leak source code which could have security issues.

And the sources aren't available in prod anyway is it ? how will it get the sources?

Reply:
Good point. I changed to Dev Mode only.